entity-types/ext-sentrysoftware/definition.yml

```diff
@@ -0,0 +1,24 @@
+domain: EXT
+type: SENTRYSOFTWARE
+synthesis:
+  rules:
+  - identifier: agent.host.name
```

Comment:
Are you sure that the name will be enough to identify the entity? Let me set an example: If the account contains two sentry software with the same `agent.host.name`, it will show only one entity in New Relic.